build_resolvers/lib/src/resolver.dart

```diff
@@ -249,3 +254,38 @@ Future<void> _createSdkVersionFile(File sdkVersionFile) async {
   await sdkVersionFile.create(recursive: true);
   await sdkVersionFile.writeAsString(Platform.version);
 }
+
+List<int> buildSdkSummary(String dartSdkPath) {
+  // Prepare the Dart SDK.
+  var resourceProvider = PhysicalResourceProvider.INSTANCE;
+  var dartSdkFolder = resourceProvider.getFolder(dartSdkPath);
+  var dartSdk = FolderBasedDartSdk(resourceProvider, dartSdkFolder, true)
+    ..useSummary = false
+    ..analysisOptions = AnalysisOptionsImpl();
+
+  // Try to load the flutter engine _embedder.yaml file and add any missing
+  // libraries to the dart sdk.
+  var embedderSdkPath =
+      p.join(p.dirname(dartSdkPath), 'pkg', 'sky_engine', 'lib');
+  var embedderYamlFile =
+      resourceProvider.getFile(p.join(embedderSdkPath, '_embedder.yaml'));
```

Comment:
Does `resourceProvider` expect platform specific directory separators or should these be normalized with `p.url`?

Reply:
Once I figure out how to write a test we will find out ;)

Reply:
Looks like this works!